index.html

```diff
@@ -20,13 +20,19 @@
   </head>
   <body>
```

Comment:
[ Required ] Good job, your desktop version is responsive. However, I noticed that in devices with widths `376px` to `758px` the layout of your application is not consistent with the Figma template provided this is because you've applied a width of `375px` on these devices thus causing a gray margin to the right of the screen. Please see the screenshot below and fix this as it will improve your project.

- [ ] You can fix this by applying a width of `100%` for all elements that have a width of `375px` e.g
```CSS
.about-contact, .about, .nav, .main, .workSection {
  width: 100%;
}

.card {
  width: 90%;
  margin: 0 auto;
}

.card .img {
    background-size: 100%;
    width: 90%;
    height: 400px;
    border-radius: 8px;
    margin: 16px;
}
```

## Your current implementation
![Screenshot from 2022-11-04 22-58-19](https://user-images.githubusercontent.com/16392046/200065096-35f97439-34fc-480b-bb7f-f993daaae3df.png)